### ISSUE TITLE
[Bug]: Select trigger is not correctly disabled

### ISSUE THREAD

**Issue by misbahansori** [radix-vue/shadcn-vue#354]
### Reproduction

https://codesandbox.io/p/devbox/shadcn-vue-selectdemo-8tzj6j?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clstrc2i200073b6aun146aq4%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clstrc2i200023b6aps6eevwk%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clstrc2i200043b6atsub9cwq%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clstrc2i200063b6awqanyorw%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clstrc2i200023b6aps6eevwk%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clstrc2i200013b6aqa6783pq%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fcomponents.json%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522clstrc2i200023b6aps6eevwk%2522%252C%2522activeTabId%2522%253A%2522clstrc2i200013b6aqa6783pq%2522%257D%252C%2522clstrc2i200063b6awqanyorw%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clstrc2i200053b6ajvkgoh4y%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522port%2522%253A5173%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clstrc2i200063b6awqanyorw%2522%252C%2522activeTabId%2522%253A%2522clstrc2i200053b6ajvkgoh4y%2522%257D%252C%2522clstrc2i200043b6atsub9cwq%2522%253A%257B%2522id%2522%253A%2522clstrc2i200043b6atsub9cwq%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clstrc2i200033b6a7otigbaf%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clstrc2i200033b6a7otigbaf%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### Describe the bug

The disabled select does not have the same style as the disabled input
![Screenshot 2024-02-20 at 09 44 32](https://github.com/radix-vue/shadcn-vue/assets/20674057/77661a02-252f-4d84-a9ad-7652ed0c2ec6)
upon inspection, the select root has disabled set to true, but the select trigger does not

![Screenshot 2024-02-20 at 09 47 14](https://github.com/radix-vue/shadcn-vue/assets/20674057/1b97fb52-fe8f-4f73-b151-ffcfe9980bc5)
![Screenshot 2024-02-20 at 09 47 20](https://github.com/radix-vue/shadcn-vue/assets/20674057/8851d5e5-219a-4867-9f5f-3ebbfccd3bed)

### System Info

```bash
System:
    OS: macOS 14.3.1
    CPU: (8) arm64 Apple M1
    Memory: 73.80 MB / 8.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/.nvm/versions/node/v20.10.0/bin/node
    Yarn: 1.22.10 - /opt/homebrew/bin/yarn
    npm: 10.2.3 - ~/.nvm/versions/node/v20.10.0/bin/npm
    pnpm: 8.15.3 - ~/Library/pnpm/pnpm
    bun: 1.0.0 - ~/.bun/bin/bun
  Browsers:
    Brave Browser: 121.1.62.165
    Chrome: 121.0.6167.184
    Safari: 17.3.1
  npmPackages:
    @vueuse/core: ^10.7.2 => 10.7.2 
    nuxt: ^3.10.0 => 3.10.0 
    radix-vue: ^1.3.2 => 1.3.2 
    shadcn-nuxt: ^0.8.6 => 0.8.7 
    vue: ^3.4.15 => 3.4.15
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by sadeghbarati** [comment]
~~Related https://github.com/radix-vue/shadcn-vue/issues/333~~

Can you try radix-vue **`v1.4.0+`**?

**Comment by misbahansori** [comment]
> Related #333
> 
> Can you try radix-vue **`v1.4.0+`**?

Yup, I've tried the `"radix-vue": "^1.4.7"` and the issue still exists

**Comment by sadeghbarati** [comment]
I think the PR I mentioned will fix this issue 

for the workaround, you can add disabled to

1. `Select` (SelectRoot)
2. `SelectTrigger`

**Comment by sadeghbarati** [comment]
Fixed in radix-vue `v1.4.8`

https://stackblitz.com/edit/hru8yz?file=src%2FApp.vue
